kolibri/content/api.py

```diff
@@ -28,6 +28,11 @@ class ChannelMetadataFilter(filters.FilterSet):
     available = filters.django_filters.MethodFilter()
 
     def filter_available(self, queryset, value):
+        if value == "True":
```

Comment:
yuck - when does something get set to the string `"True"`?

Reply:
When I added "?available=True" to the url (`/api/channel/`), the value was parsed as string "True" instead of boolean value True... So I had to change it to boolean value True here.

Reply:
@lyw07 note that Javascript `true` is not `True` but `true`. But, I believe you will still need the string check unfortunately.

Reply:
Ah thank you! So I think I'll have to change it to `value == "true"` instead?

Reply:
Yeah - I dislike the ickiness of it as much as @indirectlylit does, but for some reason there is no GET Param standard for Booleans.

Reply:
one pattern I've seen is empty string for false and anything else for true:

* false `?available=`
* true `?available=1`
* true `?available=true`
* true `?available=True`
* true `?available="green plants are nice"`

Reply:
that's maybe a confusing API though. sometimes more straightforward just to use `0` and `1`

Reply:
Note that our js api client will set a js true to the string of the same and false to string false, so I would prefer to follow that convention.

Reply:
I just checked `learn.html` (if I find it correctly as the learn page. Sorry I don't know much about front-end), it's using the boolean value `True` to pass into the api call. Maybe we can define that `"true"`, `"True"`, `True`, `1`  stand for `True`, otherwise they are false?